sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 5.7.0-beta.2 (Unreleased)
-
-### Features Added
```

Comment:
I removed the entry for idempotent publishing per PR suggestion but now find it weird with just a sample entry for a new beta...

Reply:
perfect!

Reply:
Updated